dnf/cli/cli.py

```diff
@@ -808,7 +808,8 @@ def _process_demands(self):
 
         if demands.root_user:
             if not dnf.util.am_i_root():
-                raise dnf.exceptions.Error(_('This command has to be run under the root user.'))
+                raise dnf.exceptions.Error(
+                        _('This command has to be run under fully privileged user (e.g. root).'))
```

Comment:
Shall rather tell me how many spaces is OK.

Reply:
It would be cool if Sider told us how to fix the style. And then it could also fix it for us :slightly_smiling_face: 

PEP8. 4 spaces more than the previous indent.

Reply:
I don't think that this is good idea. See implementation in `dnf.util.am_i_root()`:
`return os.geteuid() == 0`

I believe that the original statement is more correct that the new one. Or we have to change the test in `am_i_root()`.

Reply:
It will look terribly in this particular case, but whatever.

Reply:
Ok, made it consistent with line 278.  Slightly better in my view.